guardrails/guard.py

```diff
@@ -496,6 +500,14 @@ def from_pydantic(
         guard._exec_opts = exec_opts
         guard._output_type = schema.output_type
         guard._base_model = output_class
+        if isinstance(output_formatter, str):
+            if isinstance(output_class, list):
```

Comment:
We do support root level lists for other flows.   Is this restriction because of a limitation with JSONFormer?

Reply:
Yes. :(

Reply:
Fair enough, let's make the error message more specific then to explicitly state that output_formatter="jsonformer" can't be used with root-level lists.

Reply:
btw root-level lists are apparently an anti-pattern from an OpenAI function calling perspective - they only support top-level object for structured data

Reply:
It turns out they're valid JSON, though!  I did not know that.

Reply:
> btw root-level lists are apparently an anti-pattern from an OpenAI function calling perspective - they only support top-level object for structured data

@zsimjee 
Is this a new limitation with `tools` vs the old `functions` args?  We've been using function calling with top level arrays with that syntax for a while now